src/OrchardCore.Modules/OrchardCore.Redis/Startup.cs

```diff
@@ -56,6 +66,11 @@ public override void ConfigureServices(IServiceCollection services)
             services.AddSingleton<IModularTenantEvents>(sp => sp.GetRequiredService<IRedisService>());
             services.AddSingleton<IRedisDatabaseFactory, RedisDatabaseFactory>();
         }
+
+        // Callback for accepting any certificate as long as it exists, while ignoring other SSL policy errors.
+        // This allows the use of self-signed certificates on the Redis server.
+        private static bool AlwaysValidateCertificate(object sender, X509Certificate certificate, X509Chain chain, SslPolicyErrors sslPolicyErrors)
```

Comment:
Is the a proper name for this method? It can't be called AlwaysValidateCertificate if you don't always use it.

Maybe something like IgnoreCertificateValidation or something else

Reply:
Well, it clearly doesn't have that name because it's always used. It has that name because it's a handler that always accepts the certificate, regardless of how insecure it is. Maybe not a great name regardless, but it was hard to come up with something clever, and it's not like it's going to be used in a lot of places. But I suppose I can think about it and see if I can come up with something, hmm...

Reply:
@ovekaaven the idea is to improve the code readability in OC.

Reply:
Came up with something, done.